### DISCUSSION TITLE
Add `tea ls` command.

### DISCUSSION THREAD

**Discussion by mittalyashu** [pkgxdev/pkgx#676] [Ideas]
Add support for `ls` sub-command, to list all the existing/installed CLIs.

---

Maybe it's just me, but I am kind of lazy to do to

`~/.tea` directory to see all the CLIs installed along with it's versions.

**Comment by mxcl** [comment]
Yep agreed, I have somewhat immediate plans for a `-Q` query type parameter that can be used to inspect both the pantry and the cellar.

**Reply by mittalyashu** [reply]
Can you share an example command to get the gist of it?

**Reply by mxcl** [reply]
Added `mash pkgx/ls`, mash can be run via `pkgx`. https://mash.pkgx.sh
